p2panda-rs/src/document/document.rs

```diff
@@ -56,8 +56,8 @@ impl Document {
         &self.view_id
     }
 
-    /// Get the document author.
-    pub fn author(&self) -> &Author {
+    /// Get the document author's public key.
+    pub fn author(&self) -> &PublicKey {
```

Comment:
Maybe we should also rename the method to `public_key` (and struct fields etc.)

Reply:
Ah yeah, Author actually makes sense on a document as it could be many public keys